pallets/auctions/src/lib.rs

```diff
@@ -630,12 +663,42 @@ impl<T: Config> NftAuction<T::AccountId, T::AuctionId, BalanceOf<T>, Auction<T>>
 
 		Ok(())
 	}
+
+	fn validate_general_data(&self) -> DispatchResult {
+		Pallet::<T>::validate_general_data(&self.general_data)?;
+
+		if let Some(reserve_price) = self.general_data.reserve_price {
+			ensure!(
+				reserve_price == self.general_data.next_bid_min,
```

Comment:
thought reserve price (min amount for which a user is willing to sell) can be higher than starting bid

Reply:
not for English auctions

Reply:
(bc of bidding by bots)